applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -37,11 +39,15 @@ enum dvfs_state {
 	DVFS_STATE_INITIALIZING,
 	DVFS_STATE_LLPM_CONNECTED,
 	DVFS_STATE_USB_CONNECTED,
+	DVFS_STATE_CONFIG_CHANNEL,
+	DVFS_STATE_SMP_TRANSFER,
 	DVFS_STATE_COUNT
 };
 
 static const uint8_t dvfs_high_freq_bitmask = BIT(DVFS_STATE_USB_CONNECTED) |
-					      BIT(DVFS_STATE_INITIALIZING);
+					      BIT(DVFS_STATE_INITIALIZING) |
```

Comment:
Consider keeping order the same as for `enum dvfs_state` members to improve readability